src/lib/components/multiDropzone/UploadFileCell/index.tsx

```diff
@@ -2,7 +2,8 @@ import React from 'react';
 import classnames from 'classnames';
 
 import styles from './style.module.scss';
-import icons from '../icons/index';
```

Comment:
We can now remove these icons, right? The whole folder 👀

Reply:
Yes, thanks for catching it, we can delete them all - done ✅